src/switch_rtp.c

```diff
@@ -5130,6 +5132,16 @@ SWITCH_DECLARE(switch_status_t) switch_rtp_create(switch_rtp_t **new_rtp_session
 		}
 	}
 
+	rtp_session->poll_timeout_s = TELNYX_RTP_DEFAULT_POLL_TIMEOUT_S;
+
+	{
+		const char *v = switch_channel_get_variable(channel, "telnyx_rtp_poll_timeout_s");
+		if (!zstr(v) && switch_true(v)) {
```

Comment:
Under the function switch_true it is also calling atoi to validate if its true and that is the only validation that we need it here. Passing accidentally like `t` will make this return true then poll_timeout_s will be having a 0 value. So either remove this then validate the result of atoi if its greater than 0 or replace it with this condition
```
switch_is_number(v) && atoi(v)
```

Reply:
This is a good catch. Thank you.

Reply:
We should replace switch_true(v) here with switch_is_number(v)